DB/src/main/java/io/deephaven/db/v2/select/DhFormulaColumn.java

```diff
@@ -497,17 +510,22 @@ private CodeGenerator generateFillChunk(boolean usePrev) {
     @NotNull
     private CodeGenerator generateFillChunkHelper(TypeAnalyzer ta) {
         final CodeGenerator g = CodeGenerator.create(
-                "private void fillChunkHelper(final FormulaFillContext __context,", CodeGenerator.indent(
+                "private void fillChunkHelper(final boolean __usePrev, final FormulaFillContext __context,", CodeGenerator.indent(
                         "final WritableChunk<? super Attributes.Values> __destination,",
                         "final OrderedKeys __orderedKeys[[ADDITIONAL_CHUNK_ARGS]])"), CodeGenerator.block(
                         "final [[DEST_CHUNK_TYPE]] __typedDestination = __destination.[[DEST_AS_CHUNK_METHOD]]();",
-                        CodeGenerator.optional("maybeCreateI",
-                                "__context.__iChunk.setSize(0);",
-                                "__index.invert(__orderedKeys.asIndex()).forAllLongs(l -> __context.__iChunk.add(__intSize(l)));"
-                        ),
-                        CodeGenerator.optional("maybeCreateII",
-                                "__index.invert(__orderedKeys.asIndex()).fillKeyIndicesChunk(__context.__iiChunk);"
-                        ),
+                        CodeGenerator.optional("maybeCreateIOrII",
+                                "try (final Index prev = __usePrev ? __index.getPrevIndex() : null)", CodeGenerator.block(
+                                        "try (final Index inverted = ((prev != null) ? prev : __index).invert(__orderedKeys.asIndex()))", CodeGenerator.block(
```

Comment:
No need to nest these. Use a semicolon to put them in the same block.

Reply:
That was intentional.  Given the little control we get from the CodeGenerator, I like the readability/indentation result of what I had better.  But, suit yourself, I don't care enough to argue.

Reply:
I don't, either. If the code gen makes the other way look bad, keep this.